web-app/src/main/java/com/infoshare/webapp/service/QuestionService.java

```diff
@@ -94,36 +101,39 @@ public static void removeQuestion(List<Questions> questionsList) throws IOExcept
         questionsList.remove(questionToRemove);
     }
 
-    private static void showAvailableQuestions(List<Questions> questionsList){
-        for (Category cat: Category.values()) {
+    private static void showAvailableQuestions(List<Questions> questionsList) {
+        for (Category cat : Category.values()) {
             System.out.println("Kategoria: " + cat);
             System.out.print("Dostępne numery pytań: ");
             StringBuilder stringBuilder = new StringBuilder();
 
-            questionsList.stream().filter(c -> c.getCategory().equals(cat)).forEach(s -> stringBuilder.append(s.getIdQuestion()+", "));
+            questionsList.stream().filter(c -> c.getCategory().equals(cat)).forEach(s -> stringBuilder.append(s.getIdQuestion() + ", "));
             System.out.println(stringBuilder.deleteCharAt(stringBuilder.lastIndexOf(",")));
         }
         // return all possible number question to choose
         availableQuestionNumbers = questionsList.stream().map(Questions::getIdQuestion).toList();
     }
 
-    private static void showQuestion(Questions questions){
+    private static void showQuestion(Questions questions) {
         System.out.println("Wybrałeś pytanie: ");
         System.out.println(questions.toString());
     }
```

Comment:
to już chyba nie jest potrzebne

Reply:
Jest osobny task na wyrzucenie starych metod